### PR TITLE
Add examples for z_rpc calls

### LINKED ISSUE #2723: Add curl example to z_sendmany help output
Some of the z_addr rpc calls do not have example curl commands. Am adding an example to ZIG page, would be good to add to output of help option on the calls themselves.

### PR DESCRIPTION
**Pull request by arcalinea**
Adds missing cli and json-rpc examples for `z_` rpc calls

Closes #2723

### CODE REVIEW

**Comment by bitcartel** [comment]
Can you squash the commits?

**Comment by arcalinea** [comment]
@bitcartel latest commit addresses #2777

**Comment by str4d** [comment]
@zkbot r+

**Comment by str4d** [comment]
Temporary failure downloading libsodium from the upstream server. I've cached it on our servers.

@zkbot retry

File: src/wallet/rpcwallet.cpp [hunk 1]
```diff
@@ -3542,12 +3554,15 @@ UniValue z_shieldcoinbase(const UniValue& params, bool fHelp)
...
             "\nResult:\n"
             "{\n"
-            "  \"operationid\": xxx          (string) An operationid to pass to z_getoperationstatus to get the result of the operation.\n"
-            "  \"shieldedUTXOs\": xxx        (numeric) Number of coinbase utxos being shielded.\n"
-            "  \"shieldedValue\": xxx        (numeric) Value of coinbase utxos being shielded.\n"
             "  \"remainingUTXOs\": xxx       (numeric) Number of coinbase utxos still available for shielding.\n"
             "  \"remainingValue\": xxx       (numeric) Value of coinbase utxos still available for shielding.\n"
+            "  \"shieldingUTXOs\": xxx        (numeric) Number of coinbase utxos being shielded.\n"
+            "  \"shieldingValue\": xxx        (numeric) Value of coinbase utxos being shielded.\n"
+            "  \"opid\": xxx          (string) An operationid to pass to z_getoperationstatus to get the result of the operation.\n"
```
Comment (daira) [inline src/wallet/rpcwallet.cpp]: Oh, I hadn't realized that we were inconsistent in the naming of this field between different operations. That's unfortunate.

**Pull request merged**